ezpublish/config/ezpublish_behat.yml

```diff
@@ -1,46 +1,2 @@
-doctrine:
-    dbal:
-        connections:
-            default:
-                driver: pdo_mysql
-                host: localhost
-                user: root
-                dbname: behattestdb
-                charset: UTF8
-
-ezpublish:
-    repositories:
-        behat: ~
-    siteaccess:
-        # Available siteaccesses
-        list:
-            - behat_site
```

Comment:
Which siteaccess are you running behat tests with now?
think this is good cleanup, as I don't mind avoiding having to hardcode behat siteaccess in default data.